lib/wlib/utilities/Comparator.h

```diff
@@ -138,31 +138,31 @@ namespace wlp {
      * @tparam tSize static string size
      */
     template<uint16_t tSize>
-    struct Comparator<StaticString<tSize>> {
```

Comment:
ah you hit autoformat on this file and clion broke the formatting (because it's glitchy sometimes) we can fix this later

Reply:
bruhh rip I thought it would be fine. I will fix some other time